### PR TITLE
Webob

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

**Comment by stuertz** [comment]
@conda-forge/core It is ready for merge.

File: recipes/webob/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: webob
+  version: "1.6.2"
+
```
Comment (ocefpaf) [inline recipes/webob/meta.yaml]: Please see the example recipe and use a jinja variable for the version.

File: recipes/webob/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  md5: cc88bf68d6bf879c2d2145c86b6f5010
+
+build:
+  number: 0
+  script: python setup.py install 
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (ocefpaf) [inline recipes/webob/meta.yaml]: If the package uses setuptools you should add the extra args in the script call on line 12.

Comment (stuertz) [inline recipes/webob/meta.yaml]: Thanks for the review. I've updated the PR

File: recipes/webob/meta.yaml (hunk 1 above)
Comment (stuertz) [inline recipes/webob/meta.yaml]: done.

**Pull request merged**